src/convert_image/x86.rs

```diff
@@ -16,6 +16,38 @@
 
 use crate::convert_image::common::*;
 
+#[cfg(target_arch = "x86")]
+use core::arch::x86::_bswap;
+#[cfg(target_arch = "x86_64")]
+use core::arch::x86_64::_bswap;
+#[cfg(target_arch = "x86_64")]
+use core::arch::x86_64::_bswap64;
+
+#[cfg(target_arch = "x86_64")]
+#[inline(always)]
```

Comment:
I think rust is able to inline automatically this function in release.
In this case it is preferable to leave the function without the inline.